tests/schemer/sample.py

```diff
@@ -36,6 +36,16 @@ def get_author_schema(document):
     else:
         return name_schema
 
+website_schema = Schema({
+    "url": {"type": basestring, "required": True},
+    "name": {"type": basestring, "required": True}
+    })
+
+def get_website_schema(document):
+    if isinstance(document, list):
+        return list
```

Comment:
Maybe return an `Array` here which validates each member adheres to the `website_schema`??